src/web/app/pages-instructor/instructor-courses-page/__snapshots__/instructor-courses-page.component.spec.ts.snap

```diff
@@ -11,6 +11,7 @@ exports[`InstructorCoursesPageComponent should snap when it is undeletable and u
   courseService={[Function CourseService]}
   courseStats={[Function Object]}
   httpRequestService={[Function HttpRequestService]}
+  isAddNewCourseFormExpanded="undefined"
```

Comment:
Take note that if query param is not present, it's undefined.

Reply:
Updated it to only set `isAddNewCourseFormExpanded` is query param is not undefined so that it's more explicit